_posts/2024-02-04-opa-arrives.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title:  "Open Policy Agent for Trino arrived"
+author: Manfred Moser
+image: /assets/images/logos/opa-small.png
+excerpt_separator: <!--more-->
+show_pagenav: true
+---
+
+Trino now ships with an access control integration using the popular and widely
+used [Open Policy Agent (OPA)](https://www.openpolicyagent.org/) from the Cloud Native
+Computing Foundation. The release of [Trino
+438]({{site.url}}/docs/current/release/release-438.html) marks an important
+milestone of the effort towards this integration.
+
+<!--more-->
+
+## Collaboration and history
+
+Open Policy Agent was first released in 2016 and has gained more and more
+popularity in the ecosystem of cloud native application and beyond.
+
+Initial efforts for an integration with Trino started at Bloomberg, Stackable,
+Raft and others places separately, in parallel and partially in collaboration.
+You first heard about it more in August 2022 in the [Trino Community Broadcast
+episode 39]({{site.url}}/episodes/39.html) with a team from Raft as guests.
+
+Usage and experience with OPA grew and in the end Pablo Arteaga from
+[Bloomberg](https://www.bloomberg.com) and Sönke Liebau from
```

Comment:
```suggestion
Usage and experience with OPA grew. In the end, Pablo Arteaga from
[Bloomberg](https://www.techatbloomberg.com/) and Sönke Liebau from
```

Minor reword? And could we use https://www.techatbloomberg.com/ instead of Bloomberg.com?